mteb/tasks/Classification/eng/FinancialPhrasebankClassification.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskClassification
+
+
+class FinancialPhrasebankClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="FinancialPhrasebankClassification",
+        description="Polar sentiment dataset of sentences from financial news, categorized by sentiment into positive, negative, or neutral.",
+        reference="https://arxiv.org/abs/1307.5336",
+        dataset={
+            "path": "financial_phrasebank",
+            "revision": "1484d06fe7af23030c7c977b12556108d1f67039",
+            "name": "sentences_allagree",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["train"],
+        eval_langs=["eng-Latn"],
+        main_score="accuracy",
+        date=None,
```

Comment:
Can just provide a rough estimate here

Reply:
I didn't see any dates on huggingface or in the paper. I have no way of estimating it. Also I see a lot of [other datasets](https://github.com/embeddings-benchmark/mteb/blob/edef77bbcce8e0a7a362627b13596391d1a4c3d9/mteb/tasks/Classification/eng/Banking77Classification.py#L22) have set it to None.